### DISCUSSION TITLE
Thank you for this package

### DISCUSSION THREAD

**Discussion by zehawki** [react-page/react-page#905] [General]
Just a note to express my sincere thanks to the folks who built this, contributed and now maintain this. Its quite an extraordinary project. Most libraries / packages / whatchmacallit have dozens of choices. Whereas for block based page builders that are extendible, and MIT licensed - there is only 1. This one.

**Comment by macrozone** [comment]
we have to thank @arekkas who initialy built it and open sourced it! I was quite happy to find something like this and I had great success with using it in comercial apps for some time now.

**Reply by zehawki** [reply]
@macrozone once again, a big thank you to you for the zoom functionality. That was sorely needed! It makes life so much easier when designing complex pages.
